README.md

```diff
@@ -153,6 +153,41 @@ then
 curl http://localhost:9210/care_supervision_epo_cases-000001/_search
 ```
 
+## Uploading ccd definition into PR environment
+On PR env following ccd definition files are generated and stored as jenkins job artefacts:
+- ccd-fpl-preview-<PR-ID>-toggle-on.xlsx (uploaded automatically by jenkins)	
```

Comment:
the rich diff doesn't seem to be displaying anything here however the markdown interpreter in intellij is 🤔 
![image](https://user-images.githubusercontent.com/22620804/105381202-aa339b00-5c06-11eb-93fa-8d099d35f654.png)
![image](https://user-images.githubusercontent.com/22620804/105381277-bfa8c500-5c06-11eb-8642-9dc1de1910cd.png)

Reply:
done